.github/workflows/build_docker.yml

```
@@ -29,4 +29,4 @@ jobs:
           context: .
           file: ./Dockerfile
           push: true
-          tags: ${{ secrets.DOCKERHUB_USERNAME }}/tools:nightly
\ No newline at end of file
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/openneuro_uploader_api:nightly
```

```suggestion
          tags: ${{ secrets.DOCKERHUB_USERNAME }}/openneuro_upload:nightly
```
:cherries: How about this? already pretty long to type